src/api/libcellml/annotator.h

```diff
@@ -675,22 +675,20 @@ class LIBCELLML_EXPORT Annotator: public Logger
     /**
      * @brief Assign an automatically generated, unique identifier to the given @p model.
      *
+     * @overload
+     *
      * Assign an automatically generated, unique identifier to the given @p model.
      * The default is to assign an identifier to a MODEL item.
      * This method will return the new identifier that has been assigned, or an empty string
      * if the operation failed.
      *
      * Only CellmlElementType::MODEL and CellmlElementType::ENCAPSULATION are relevant any other
-     * CellmlElementType type will not assign an identifier.  @see errorCount() if and/or error(size_t) for any
-     * issues that may have been raised.
-     *
-     * @see assignId(const AnyCellmlElementPtr &item) for a list of reasons that an identifier may not be assigned.
-     * @see errorCount() if and/or error(size_t) for any issues that may have been raised.
+     * CellmlElementType type will not assign an identifier.
```

Comment:
Might be good to have some `@ref` here and there (e.g., for `CellmlElementType` on line 686).

Reply:
a7ccc63